db/migrate/20240311151946_add_trust_fund_to_capital.rb

```diff
@@ -0,0 +1,7 @@
+class AddTrustFundToCapital < ActiveRecord::Migration[7.0]
+  def change
+    add_column :capitals, :will_benefit_from_trust_fund, :string
+    add_column :capitals, :trust_fund_amount_held, :integer
+    add_column :capitals, :yearly_dividend, :integer
```

Comment:
Should this be renamed trust_fund_yearly_dividend as it is stored on capital and other things could have a yearly_dividend?